cdi/embedded/src/test/resources/infinispan.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
```

Comment:
I think that we should probably use resource filtering for many of these configuration files and use the infinispan.core.schema.version property which lives in the bom

Reply:
this is a task for @ryanemerson

Reply:
Isn't it possible just to point the IDE to the schema in `target/` opposed to `src/`, that way it will be a valid `.xsd` file. Granted you will have to run `mvn install` first.